templates/rust/common.tera

```diff
@@ -668,12 +877,21 @@ where
             marker: PhantomData,
         }
     }
+
+    /// Get mask for bitfield, mask is unshifted and at the actual offset
```

Comment:
I think this should be removed `and at the actual offset`. At least it seems confusing

Reply:
same as above^^